docs/src/appendix/forge-library/read_json.md

```diff
@@ -0,0 +1,47 @@
+# `read_json`
+
+> `fn read_json(file: @File) -> Array<felt252>`
+
+Read and parse json file content to an array of felts.
+
+- `file` - a snapshot of an instance of the struct `File` that consists of the following fields:
+  - `path` - Cairo shortstring representing a path to a file relative to a package root.
+
+```rust
+use snforge_std::{ FileTrait, read_json };
+
+#[test]
+fn test_read_json() {
+    let file = FileTrait::new('data/file.json');
+    let content = read_json(@file);
+    // ...
+}
+```
+
+- File content must have proper JSON Format with values satisfying the conditions:
+  - integers in range of `[0, P)` where P is [`Cairo Prime`](https://book.cairo-lang.org/ch02-02-data-types.html?highlight=prime#felt-type)
+  - strings of length `<=31`
+  - array of integers or strings fulfilling the above conditions
+
+For example, this file content:
+```json
+{
+    "b": 1,
+    "a": 12,
+    "d": {
+        "e": 1234
+    },
+    "c": "123"
+}
+```
+will be parsed to the following array:
+> ⚠️ **Warning**
```

Comment:
I think it should be higher on the page, just like in `parse_json`. Please address other comments from `parse_json` here as well